env.ts

```diff
@@ -0,0 +1,4 @@
+export default {
```

Comment:
Are we going to save this directly in the repo or should we use real `.env`?

Reply:
I understand this refactor. My question alludes as the whether we should expose this kind of data to the repo or not.
And if we should use `.env` that adds the variables to the `process.env` of Node or not. 💮